gulpfile.js

```diff
@@ -0,0 +1,33 @@
+const gulp = require('gulp')
+const gutil = require('gulp-util')
+const gclean = require('gulp-clean')
+const typescript = require('gulp-typescript')
+const sourcemaps = require('gulp-sourcemaps')
+const tsProject = typescript.createProject('tsconfig.json')
+
+const OPTIONS = {
+  dirs: {
+    sources: './src',
+    build: './lib'
```

Comment:
I'm not sure, I've seen both and I'm not sure that we came to a decision. What I've been doing is just continuing on as-is and then when we've finalised the TS processes I'll update everything. That being said, I don't care either way, so I'll just change it now.